packages/tests/tests/filter/subscribe.node.spec.ts

```diff
@@ -293,6 +293,38 @@ const runTests = (strictCheckNodes: boolean): void => {
       });
     });
 
+    it("Subscribe to 30 topics in separate streams (30 streams for Filter is limit) at once and receives messages", async function () {
```

Comment:
was trying to debug in https://github.com/waku-org/js-waku/pull/2436
with no success, disabling this test with a follow up item https://github.com/waku-org/js-waku/issues/2438